src/OpenSage.Game/Terrain/Terrain.cs

```diff
@@ -402,8 +397,7 @@ private struct BlendData
                 {
                     if (tgaFile.Header.Width != largestTextureSize)
                     {
-                        tgaImage.Mutate(x => x
-                            .Resize((int) largestTextureSize, (int) largestTextureSize, MapTextureResampler));
+                        tgaImage.Mutate(x => x.Resize((int) largestTextureSize, (int) largestTextureSize));
```

Comment:
What happened to the overload that takes an IResampler? I needed to use that because the default one resulted in low quality resampling.

Reply:
For me it looked the same with the default sampler, maybe there was a bug in a previous version?

Reply:
The problem was only visible on terrain textures that were smaller than the standard size. I can’t remember if they were only used on certain maps. I need to check.

Reply:
I fixed it now. The resamplers are static members in ImageSharp itself now